app/controllers/static_pages_controller.rb

```diff
@@ -9,4 +9,7 @@ def contact
 
   def dashboard
   end
+
+  def settings
+  end
```

Comment:
@lalusaud - Why this blank method?

Reply:
because I just want to display a settings link. Don't worry I will change that later.